service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -85,11 +85,11 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
     runSetDao.createRunSet(runSet);
 
     // Fetch the entity from WDS:
-    EntityResponse entityResponse;
+    RecordResponse recordResponse;
```

Comment:
Same comment about `records` here from above. Should this be names `WdsRecordResponse`?

Reply:
Same thing here. I didn't realize `RecordResponse` is class from WDS library. It makes sense for the variable name to be `recordResponse` to be consistent.